src/models/generated/AC1A.jl

```diff
@@ -189,46 +189,46 @@ get_states_types(value::AC1A) = value.states_types
 get_internal(value::AC1A) = value.internal
 
 """Set AC1A Tr."""
-set_Tr!(value::AC1A, val) = value.Tr = val
+set_Tr!(value::AC1A, val::Float64) = value.Tr = val
```

Comment:
Something went wrong with your rebase.  Commit ID b9fadd466 from Jose removed the types from these parameters.  Commit ID 2e996e0e0 from you put them back.

That doesn't seem related to your error message but there could be a common cause.  Can you determine what happened?

Reply:
@jd-lara and @daniel-thom, you are both right, sorry! my manifest was pinned on IS 0.13 for some reason. Fixing it now

Reply:
done tests passing locally now! Will wait for the ci.